build.yaml

```diff
@@ -0,0 +1,9 @@
+targets:
+  $default:
+    builders:
+      freezed:freezed:
+        generate_for:
+          include:
+            - lib/people/model/**.dart
```

Comment:
You can make a model folder for every feature and include it here like this:
`lib/**/model/**.dart`
Then you don't have to add every new feature folder with models here